core/modules/modsearcheverywhere.class.php

```diff
@@ -65,10 +65,8 @@ public function __construct($db)
         $this->description = "Description of module searcheverywhere";
         // Possible values for version are: 'development', 'experimental' or version
 
-        $this->version = '2.1.0';
-		// Url to the file with your last numberversion of this module
-		require_once __DIR__ . '/../../class/techatm.class.php';
-		$this->url_last_version = \searcheverywhere\TechATM::getLastModuleVersionUrl($this);
+        $this->version = '2.0.1';
+
```

Comment:
Ya un souci sur ta pull up
```suggestion
		$this->version = '2.1.0';
		// Url to the file with your last numberversion of this module
		require_once __DIR__ . '/../../class/techatm.class.php';
		$this->url_last_version = \searcheverywhere\TechATM::getLastModuleVersionUrl($this);

```